### ISSUE TITLE
No match for logsource

### ISSUE THREAD

**Issue by asalih** [SigmaHQ/sigma#2553]
Hello, \rules\windows\other\dns_server\win_susp_dns_config.yml does not match any logsource in powershell config due to lack of category specification. So this may cause false positives. E.g. Microsoft-Windows-BitLocker/BitLocker Management has an event with id 770 and this event is not related with what we are looking for.

Conversion;
`python sigmac.py --target powershell ..\\..\\rules\\windows\\other\\dns_server\\win_susp_dns_config.yml -c powershell`

`Get-WinEvent | where {($_.ID -eq "150" -or $_.ID -eq "770") } | select TimeCreated,Id,RecordId,ProcessId,MachineName,Message`

Please consider adding the "dns" category to the rule, if it makes sense.

https://github.com/SigmaHQ/sigma/blob/eb25fcca35410f53fcff6d60431b0df6dc890930/rules/windows/other/dns_server/win_susp_dns_config.yml#L14

**Comment by frack113** [comment]
Hi
There is allready  in the powershell.yml
https://github.com/SigmaHQ/sigma/blob/a2ceddcd65090604822dfc247739baf4d8bf9458/tools/config/powershell.yml#L46

but there is a error
```
  windows-dns-server:
    product: windows
    service: dns-server
    category: dns  <----- should not be a category here
    conditions:
      LogName: 'DNS Server
```

**Comment by asalih** [comment]
Yes, that's what I meant. Config has the category specification but the rule doesn't. 
I didn't check if any other rule uses this logsource so the first thing that came to my mind is that you should consider adding the dns category to this rule, as a suggestion ofc. 

And also _windows-dns-server-audit_ don't have category specification. So if there are no rules that uses this current logsource(with category spec) it might be good to remove.

**Comment by frack113** [comment]
Hi,
I make the fix and open a discution #2564
I have keeep `windows-dns-server-audit` service for the moment

**Comment by asalih** [comment]
I was trying to refer to `windows-dns-server-audit`, which doesn't have a category. So it might be good to remove the category from `windows-dns-server` if it is not in use by any other rule. 
Anyways thanks for the fix.
